### DISCUSSION TITLE
XYZ grid

### DISCUSSION THREAD

**Discussion by oshimoto** [vladmandic/sdnext#1476] [Q&A]
Hi! I was using the xyz script to generate a grid which took me 3 hours to render but I had forgotten to change the grid margins setting. So now I will have to start all over again.

As I understand it the xyz script makes the grid after all images are generated, so my question is can't I just run the script on the existing images somehow, without rendering them again?

**Comment by vladmandic** [comment]
not the built-in one as you cant fake inputs, but there is a helper cli - take a look at `python cli/image-grid.py --help`

**Comment by oshimoto** [comment]
Thanks! I will take a look.
